app/src/main/java/com/android/unio/ui/event/EventDetails.kt

```diff
@@ -479,7 +499,10 @@ fun EventDetailsPicturesTab(event: Event, context: Context) {
                         .clip(RoundedCornerShape(10))
                         .testTag(EventDetailsTestTags.USER_EVENT_PICTURE + item.uid)
                         .clickable {
-                          scope.launch { pagerState.scrollToPage(index) }
+                          scope.launch {
+                            eventPictures.forEach { pic -> pic.author.fetch(lazy = true) }
+                            pagerState.scrollToPage(index)
```

Comment:
This seems a bit too much a fetching right here, but as this is not a big amount of data this is really fine